docs/examples.md

```diff
@@ -93,6 +93,9 @@ Create `kubernetes secrets`  based on the previous certificates
 kubectl create secret tls minio-tls-cert --key="_wildcard.minio-tenant.svc.cluster.local-key.pem" --cert="_wildcard.minio-tenant.svc.cluster.local.pem" -n minio-tenant
 kubectl create secret tls minio-buckets-cert --key="_wildcard.myminio.minio-tenant.svc.cluster.local-key.pem" --cert="_wildcard.myminio.minio-tenant.svc.cluster.local.pem" -n minio-tenant
 kubectl create secret tls minio-hl-cert --key="_wildcard.myminio-hl.minio-tenant.svc.cluster.local-key.pem" --cert="_wildcard.myminio-hl.minio-tenant.svc.cluster.local.pem" -n minio-tenant
+
+# create a new secret for the operator certs
+kubectl create secret tls operator-ca-tls-minio-tls-cert --key="_wildcard.minio-tenant.svc.cluster.local-key.pem" --cert="_wildcard.minio-tenant.svc.cluster.local.pem" -n minio-tenant
```

Comment:
The service will always be addressed as `minio.minio-tenant.svc.cluster.local`, so better use that instead of a wildcard.

Reply:
A certificate for `*.minio-tenant.svc.cluster.local` can be used on any non-headless service in that namespace, so I think it's better to use `minio.minio-tenant.svc.cluster.local` instead. If console service should also be exposed, then it may make sense to use a wildcard certificate, but it's probably more appropriate to create a custom certificate for console (or add it as an alternative name).

There is no headless `myminio` service, so I don't think the certificate for `*.myminio.minio-tenant.svc.cluster.local` would be used for anything and can be left out. The `*.myminio-hl.minio-tenant.svc.cluster.local` is totally fine.

I'm not 100% sure if operator tries to check the headless or the non-headless service. I think operator should point at the `minio.minio-tenant.svc.cluster.local` certificate, but you may want to check that.